src/ComServerHelpers/Internal/Windows/Win32/PInvoke.cs

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+
+namespace Windows.Win32;
+
+internal partial class PInvoke
+{
+    public static readonly Guid CLSID_GlobalOptions = new(0x0000034b, 0x0000, 0x0000, 0xC0, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x46);
```

Comment:
Make this use an RVA field (copy the pattern from CsWinRT).

Reply:
done